meson.build

```diff
@@ -31,16 +31,18 @@ executable(
     gresource,
     'src/Application.vala',
     'src/CategoryView.vala',
-    'src/IconView.vala',
     'src/MainWindow.vala',
-    'src/Widgets/IconListRow.vala',
+    'src/Services/Collection.vala',
+    'src/Widgets/IconListChild.vala',
     'src/Widgets/SidebarRow.vala',
+    'src/Dialogs/IconView.vala',
```

Comment:
Don't think it is a rule but it seems like we try to keep the source file listing in alpha order, so I wold move this up